lib/litmus_paper/agent_check_handler.rb

```diff
@@ -1,8 +1,19 @@
 module LitmusPaper
   class AgentCheckHandler
     def self.handle(service)
+      @cache ||= LitmusPaper::Cache.new(
+        LitmusPaper.cache_location,
+        "litmus_cache",
+        LitmusPaper.cache_ttl
+      )
       output = []
-      health = LitmusPaper.check_service(service)
+
+      health = @cache.get(service)
+      @cache.set(
+        service,
+        health = LitmusPaper.check_service(service)
+      ) unless health
```

Comment:
I think this would be easier to read with an `if ! health`, rather than a trailing unless

Reply:
I had copied and pasted from elsewhere. :)
I'll fix!